File: CodeForces/1-100/1/A/1A.py
# Problem 1A: Theatre Square
# https://codeforces.com/contest/1/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: June 14th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

import math

length, width, tile_size = str(input()).split()

print(math.ceil(int(length)/int(tile_size)) * math.ceil(int(width)/int(tile_size)))

File: CodeForces/1-100/1/A/1A.java
// Problem 1A: Theatre Square
// https://codeforces.com/contest/1/problem/A
// Compiled with: openjdk 25 2025-09-16
// Executed with: javac 25
// Linux version: Linux kernel 6.17.0-6-generic
// Submitted on: October 27th, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double n = sc.nextDouble();
        double m = sc.nextDouble();
        double a = sc.nextDouble();
        sc.close();

        long tiles_high = (long) Math.ceil(n/a);
        long tiles_wide = (long) Math.ceil(m/a);

        System.out.println(tiles_high*tiles_wide);
    }
}
